wandb/env.py

```diff
@@ -161,6 +161,12 @@ def is_debug(default: Optional[str] = None, env: Optional[Env] = None) -> bool:
     return _env_as_bool(DEBUG, default=default, env=env)
 
 
+def is_offline(env: Optional[Env] = None) -> bool:
+    if env is None:
+        env = os.environ
+    return env.get(MODE, "") == "offline"
```

Comment:
nit: `.get()` defaults to `None` by default
```suggestion
    return env.get(MODE) == "offline"
```

Reply:
If it returns `None` the equal operator will lead to undefined behaviour, hence my explicit return of empty string "".

Reply:
Totally optional -- TBH there's a benefit (extra clarity) to the more explicit way it's written here, too

Tangential: FWIW, Python is (some would say overly) lax on equality comparisons between different types -- `None == "offline"` should always evaluate to false.

For that matter, `None == anything` should evaluate to false unless
- `anything is None` or
- someone's added bespoke comparison logic under the hood (by overriding the magic method `anything.__eq__()`)

Reply:
There's nothing undefined about the equality check there, that's a pretty common pattern and I'd agree with Tony's nit. For the other places where `get` is given a default it's because the default is a meaningful value that's being returned to the caller.